sdk/communication/Azure.Communication.CallAutomation/src/Models/TransferToExternalNumber.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+using Azure.Core;
+
+namespace Azure.Communication.CallAutomation
+{
+    /// <summary>
+    /// TranferToExternalNumber
+    /// </summary>
+    [CodeGenModel("TransferToExternalNumber", Usage = new string[] { "output" }, Formats = new string[] { "json" })]
```

Comment:
is this used anywhere? cannot seem to find any that reference this

Reply:
Removed! Was a leftover from a previous version